api/handler/config.go

```diff
@@ -2,6 +2,24 @@ package handler
 
 import "net/http"
 
+type Contact struct {
+	Type  string `json:"type" example:"telegram"`
```

Comment:
Maybe we should prefix these types with `Example` so that nobody meistake them for actual types used in code?

Reply:
Or maybe change code, so that it uses actual types instead of passing `[]byte` through the entirety of the call stack? Maybe not in this PR, but later — let's make a task

Reply:
Okay, I've renamed it, I'll do the task soon